### PR TITLE
[NFC][LLVM] Add RealtimeSanitizer LLVM code owners

### PR DESCRIPTION
**Pull request by cjappl**
Split from #100596

### CODE REVIEW

**Comment by cjappl** [comment]
CC Reviewers @vitalybuka @MaskRay @davidtrevelyan 

Because we don't have commiter access, not sure if this should just be closed! I defer to the reviewers if adding this in is appropriate or not.

Thanks as always.

**Review by MaskRay — APPROVED** [review]

File: llvm/CODE_OWNERS.TXT
Comment (MaskRay) [inline llvm/CODE_OWNERS.TXT:268]: It's better to use an email address for contribution instead of `realtime.sanitizer`, which might forward to you, but are not associated with any commits.

Comment (cjappl) [reply llvm/CODE_OWNERS.TXT]: Fixed in the latest version!

**Review by vitalybuka — APPROVED** [review]

**Comment by vitalybuka** [comment]
> CC Reviewers @vitalybuka @MaskRay @davidtrevelyan
> 
> Because we don't have commiter access, not sure if this should just be closed! I defer to the reviewers if adding this in is appropriate or not.
[> 
> Thanks as always.

I don't know if this process is still relevant https://llvm.org/docs/DeveloperPolicy.html#obtaining-commit-access

**Review by MaskRay — APPROVED** [review]

**Comment by cjappl** [comment]
> > CC Reviewers @vitalybuka @MaskRay @davidtrevelyan
> > Because we don't have commiter access, not sure if this should just be closed! I defer to the reviewers if adding this in is appropriate or not.
> > [>
> > Thanks as always.
> 
> I don't know if this process is still relevant https://llvm.org/docs/DeveloperPolicy.html#obtaining-commit-access

Thanks for that link @vitalybuka! Do you think it's appropriate at this stage to apply for commit access? I have a couple commits, and only one large one (the compiler-rt work for rtsan).  With a few additional big changes needed for the first version of rtsan. 

I defer to your expertise, seeing as you're the one I've interacted the most with on PRs. I do not want to apply "too early" if it is inappropriate.

Thank you for any advice.

**Comment by vitalybuka** [comment]
> > > CC Reviewers @vitalybuka @MaskRay @davidtrevelyan
> > > Because we don't have commiter access, not sure if this should just be closed! I defer to the reviewers if adding this in is appropriate or not.
> > > [>
> > > Thanks as always.
> > 
> > 
> > I don't know if this process is still relevant https://llvm.org/docs/DeveloperPolicy.html#obtaining-commit-access
> 
> Thanks for that link @vitalybuka! Do you think it's appropriate at this stage to apply for commit access? I have a couple commits, and only one large one (the compiler-rt work for rtsan). With a few additional big changes needed for the first version of rtsan.
> 
> I defer to your expertise, seeing as you're the one I've interacted the most with on PRs. I do not want to apply "too early" if it is inappropriate.
> 
> Thank you for any advice.

https://discourse.llvm.org/t/rfc-new-criteria-for-commit-access/76290/81
Looks like LLVM it trying to limit the number of commiters https://discourse.llvm.org/t/rfc-new-criteria-for-commit-access/76290
And LLVM will likely soon going to have better commiter access requesting process.

We can try proceed as-is, without commiter, and you can apply when you think access is beneficial to you.
I assume wrong review assignees will figure out how to contact you if needed.
I suspect some OWNERS already lost commiter access.

**Comment by cjappl** [comment]
> > > > CC Reviewers @vitalybuka @MaskRay @davidtrevelyan
> > > > Because we don't have commiter access, not sure if this should just be closed! I defer to the reviewers if adding this in is appropriate or not.
> > > > [>
> > > > Thanks as always.
> > > 
> > > 
> > > I don't know if this process is still relevant https://llvm.org/docs/DeveloperPolicy.html#obtaining-commit-access
> > 
> > 
> > Thanks for that link @vitalybuka! Do you think it's appropriate at this stage to apply for commit access? I have a couple commits, and only one large one (the compiler-rt work for rtsan). With a few additional big changes needed for the first version of rtsan.
> > I defer to your expertise, seeing as you're the one I've interacted the most with on PRs. I do not want to apply "too early" if it is inappropriate.
> > Thank you for any advice.
> 
> https://discourse.llvm.org/t/rfc-new-criteria-for-commit-access/76290/81 Looks like LLVM it trying to limit the number of commiters https://discourse.llvm.org/t/rfc-new-criteria-for-commit-access/76290 And LLVM will likely soon going to have better commiter access requesting process.
> 
> We can try proceed as-is, without commiter, and you can apply when you think access is beneficial to you. I assume wrong review assignees will figure out how to contact you if needed. I suspect some OWNERS already lost commiter access.

Sounds good. I think perhaps once the main pieces of rtsan have landed and are stable, that would be a good time to apply.

**Pull request merged**
